core/markup/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "3.0.0-dev.22",
   "description": "iModel.js markup package",
   "main": "lib/imodeljs-markup.js",
+  "typings": "lib/imodeljs-markup",
```

Comment:
should this be lib/imodeljs-markup.d.ts

Reply:
It doesn't necessarily need to have the `.d.ts` extension. Typescript should pick it up either way.